atom/renderer/atom_sandboxed_renderer_client.cc

```diff
@@ -174,14 +175,20 @@ void AtomSandboxedRendererClient::DidCreateScriptContext(
                          mate::ConvertToV8(isolate, right)->ToString())));
   auto func =
       v8::Handle<v8::Function>::Cast(script->Run(context).ToLocalChecked());
+  // Load the preload script if specified
+  std::string preload_script;
+  if (!preload_script_path.empty()) {
+    base::ReadFileToString(preload_script_path, &preload_script);
```

Comment:
Check the return value of ReadFileToString for success. I think a reasonable behavior in case of failure would be to set `preload_script` to the empty string and log an error (use PLOG to get a description of the error via errno).

Reply:
@nornagon resolved